src/test/resources/missedToken.json

```
@@ -0,0 +1,22 @@
+{
+  "creationDate": "@@TODAY@@",
+  "complaint": {
+    "complaintType": "BIOMETRIC_RESIDENCE_PERMIT",
+    "reference": {
+      "referenceType": "GWF_REF",
+      "reference": "@@MISSED_TOKEN@@"
+    },
+    "reporterDetails": {
+      "applicantType": "APPLICANT",
+      "applicantName": "@@APPLICANT_NAME@@",
+      "applicantNationality": "Denmark",
+      "applicantDob": "2020-10-03",
+      "applicantEmail": "sss@uevptde.com",
```

```suggestion
      "applicantEmail": "sss@example.com",
```

Please use safely reserved domain names as specified in RFC 2606 and RFC 6761, to avoid it looking like production data as well as making sure we never accidentally email someone in a test case.

---

Used safe email and changed to the safeEmail method of Faker